### PR TITLE
Update to gradle 6.8

### PR DESCRIPTION
**Pull request by Siedlerchr**
Added hint on workaround

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,6 +50,10 @@ modularity.patchModule("test", "fastparse_2.12-1.0.0.jar")
 modularity.patchModule("test2", "fastparse-utils_2.12-1.0.0.jar")
 modularity.patchModule("test3", "sourcecode_2.12-0.1.4.jar")
 
+// Workaroud for https://github.com/openjfx/javafx-gradle-plugin/issues/89
```
Comment (stefan-kolb) [inline build.gradle:53]: typo

**Review by stefan-kolb — APPROVED** [review]

**Comment by koppor** [comment]
@Siedlerchr had the impression that JabRef was slower (see https://github.com/koppor/jabref/issues/440#issuecomment-757376716). Do we need to test it?

**Comment by Siedlerchr** [comment]
I no longer have the impression. 6.8 seems fast again with jabref or it was another bug fix

**Review by koppor — APPROVED** [review]
- `jabref.bat -n -h` works (besides the strange output https://github.com/koppor/jabref/issues/486)
- Did not see performance issues during runtime

Thus, I'll go ahead with merging. We can revert if some developers experience issues.

**Pull request merged**